unlock-js/src/FetchJsonProvider.js

```diff
@@ -1,54 +1,46 @@
 import { providers } from 'ethers'
 
-const whoAmI = global || self
-
-function getResult(payload) {
```

Comment:
this might be a weird diff thing but were we intending on removing this with this change?

Reply:
yes, this is artifacts from stuff that we do not use/need...